torch/testing/_internal/common_utils.py

```diff
@@ -63,6 +63,7 @@
 if sys.platform == 'win32':
     FILE_SCHEMA = "file:///"
 
+IS_IN_CI = bool(os.environ.get('IN_CI'))
```

Comment:
is this env consistently set to true for all ci tests?

Reply:
it should be.